packages/components/view-switcher/src/view-switcher.stories.tsx

```diff
@@ -1,56 +1,40 @@
-import type { ComponentProps } from 'react';
+import type { ReactElement } from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
-import ViewSwitcher from '.';
-import type { TViewSwitcherButtonProps } from './view-switcher-button';
 import { iconArgType, categorize, hideControls } from '@/storybook-helpers';
+import ViewSwitcher, { type TViewSwitcherProps } from '.';
 
 const BASE_CATEGORY = 'View Switcher Button';
 const BUTTONS_COUNT = 4;
 const DEFAULT_BUTTON_ICON = 'WorldIcon';
 
-type ViewSwitcherPropsAndCustomArgs = ComponentProps<
-  typeof ViewSwitcher.Group
-> & {
-  iconViewSwitcherButton1: string;
-  iconViewSwitcherButton2: string;
-  iconViewSwitcherButton3: string;
-  iconViewSwitcherButton4: string;
-  iconOnlyButton1: boolean;
-  iconOnlyButton2: boolean;
-  iconOnlyButton3: boolean;
-  iconOnlyButton4: boolean;
-  isDisabledButton1: boolean;
-  isDisabledButton2: boolean;
-  isDisabledButton3: boolean;
-  isDisabledButton4: boolean;
+type StoryAttributeName =
```

Comment:
This is the main idea that led me to writing the proposal so we don't need casting when using the `args` in the stories.